modules/ipi-install-bmc-addressing-for-fujitsu.adoc

```diff
@@ -0,0 +1,53 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id='bmc-addressing-for-fujitsu_{context}']
+
+= BMC addressing for Fujitsu
+
+The `address` field for each `bmc` entry is a URL for connecting to the {product-title} cluster nodes, including the type of controller in the URL scheme and its location on the network.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: <host name>
+        role: <master | worker>
+        bmc:
+          address: <address>
```

Comment:
I feel like this needs a footnote or something to connect it to the 'protocol' scheme statements being made above and below.